pep/app/pods/components/document/read/sidebar/template.hbs

```diff
@@ -113,16 +113,20 @@
                             @onLinkClick={{@loadDocument}}
                             @headerStickyOffset="0"
                             @document={{@selectedDocument}}
+                            @key="id"
+                            @idForFirstItem={{@selectedDocument.id}}
                         />
                     {{else}}
                         <VerticalCollection
                             @items={{@paginator.models}}
                             @tagName="ul"
+                            @key="id"
                             @containerSelector=".sidebar-content"
-                            @class="list-unstyled"
                             @estimateHeight={{if this.showHitsInContext 100 20}}
                             @renderAll={{this.fastboot.isFastBoot}}
-                            @lastReached={{@paginator.loadMoreModels}} as |result|
+                            @lastReached={{@paginator.loadMoreModels}}
+                            @idForFirstItem={{@selectedDocument.id}}
```

Comment:
@bakerac4 how does this handle when the initial page(s) of results fetched on load do not include the selected document, e.g. if its like 10 page down or something? Or was this previously handled somewhere else/or are not handling it yet?

Reply:
Thats handled somewhere else already